R/queue_didehpc.R

```diff
@@ -347,8 +352,13 @@ submit_dide <- function(obj, data, task_ids, names) {
     writeLines(glue_whisker(batch_template, list(task_id = id)), batch)
     path <- windows_path(file.path(data$paths$remote$batch, base))
     p()
+    if (length(depends_on) == 0L) {
+      deps <- character(0)
+    } else {
+      deps <- obj$dide_id(depends_on[[id]])
+    }
```

Comment:
I think Rich's previous comment was meaning this if you'd like it briefer

`deps <- if (length(depends_on) == 0L) character(0) else obj$dide_id(depends_on[[id]]`